clusterloader2/testing/load/modules/measurements.yaml

```diff
@@ -151,6 +151,27 @@ steps:
       - name: Perc50
         query: quantile_over_time(0.50, sum by (container) (container_memory_working_set_bytes / 1024 / 1024)[%v:])
 {{end}}
+  - Identifier: Quotas total usage
+    Method: GenericPrometheusQuery
+    Params:
+      action: {{$action}}
+      metricName: Quota usage
+      metricVersion: v1
+      prometheusClient: managed
+      unit: QPMs
+      queries:
+      - name: default perc99
+        query: quantile_over_time(0.99, sum by (quota_metric) (irate(serviceruntime_googleapis_com:quota_rate_net_usage{monitored_resource="consumer_quota", quota_metric="compute.googleapis.com/default"}[1m]))[%v:]) * 60
```

Comment:
Maybe we can use "dimensions" feature or "GenericPrometheusQuery" to avoid 3 copies of query?

Reply:
SG, done.